allauth/socialaccount/providers/vk/views.py

```diff
@@ -39,16 +39,11 @@ class VKOAuth2Adapter(OAuth2Adapter):
     profile_url = 'https://api.vk.com/method/users.get'
 
     def complete_login(self, request, app, token, **kwargs):
-        uid = kwargs['response']['user_id']
         resp = requests.get(self.profile_url,
                             params={'access_token': token.token,
-                                    'fields': ','.join(USER_FIELDS),
-                                    'user_ids': uid})
+                                    'fields': ','.join(USER_FIELDS)})
         resp.raise_for_status()
         extra_data = resp.json()['response'][0]
-        email = kwargs['response'].get('email')
-        if email:
-            extra_data['email'] = email
```

Comment:
Because rest-auth [passes](https://github.com/Tivix/django-rest-auth/blob/master/rest_auth/registration/serializers.py#L108) the access_token as kwargs['response'] parameter. And code `kwargs['response'].get('email')` throws exception: `{AttributeError}'str' object has no attribute 'get'`